Marlin/Configuration.h

```diff
@@ -664,6 +664,9 @@
  * readings with inductive probes and piezo sensors.
  */
 //#define PROBING_HEATERS_OFF       // Turn heaters off when probing
+#if ENABLED(PROBING_HEATERS_OFF)
+  //#define WAIT_FOR_BED_HEATER     // Wait for bed to heat back up between probes (may improve accuracy)
+#endif
```

Comment:
A PR is not complete and will not be merged if the option has not been added to all example configurations as well.